packages/twenty-server/src/workspace/workspace-migration-runner/workspace-migration-runner.service.ts

```diff
@@ -282,31 +283,55 @@ export class WorkspaceMigrationRunnerService {
     // }
   }
 
-  private async createForeignKey(
+  private async createRelation(
     queryRunner: QueryRunner,
     schemaName: string,
     tableName: string,
     migrationColumn: WorkspaceMigrationColumnRelation,
   ) {
-    await queryRunner.createForeignKey(
-      `${schemaName}.${tableName}`,
-      new TableForeignKey({
-        columnNames: [migrationColumn.columnName],
-        referencedColumnNames: [migrationColumn.referencedTableColumnName],
-        referencedTableName: migrationColumn.referencedTableName,
-        onDelete: 'CASCADE',
-      }),
-    );
-
-    // Create unique constraint if for one to one relation
-    if (migrationColumn.isUnique) {
-      await queryRunner.createUniqueConstraint(
+    if (
+      !migrationColumn.referencedSchema ||
+      migrationColumn.referencedSchema === schemaName
+    ) {
+      // We can only create a foreign key if the tables are on the same schema
+      await queryRunner.createForeignKey(
         `${schemaName}.${tableName}`,
-        new TableUnique({
-          name: `UNIQUE_${tableName}_${migrationColumn.columnName}`,
+        new TableForeignKey({
           columnNames: [migrationColumn.columnName],
+          referencedColumnNames: [migrationColumn.referencedTableColumnName],
+          referencedSchema: migrationColumn.referencedSchema,
+          referencedTableName: migrationColumn.referencedTableName,
+          onDelete: 'CASCADE',
         }),
       );
+
+      // Create unique constraint if for one to one relation
+      if (migrationColumn.isUnique) {
+        await queryRunner.createUniqueConstraint(
+          `${schemaName}.${tableName}`,
+          new TableUnique({
+            name: `UNIQUE_${tableName}_${migrationColumn.columnName}`,
+            columnNames: [migrationColumn.columnName],
+          }),
+        );
+      }
+    } else {
+      await queryRunner.query(`
```

Comment:
Not working for now, waiting for `pg_graphql` team answer

Reply:
No, because it's not supported by `pg_graphql`, see the Relationships section here: https://supabase.github.io/pg_graphql/views/

Reply:
I think we still want to have the foreign key in the DB for consistency? Will it break pg_graphql even if we don't use it that way? (but with the pg comment)